recipe/meta.yaml

```diff
@@ -1,26 +1,28 @@
 {% set name = "qstylizer" %}
-{% set version = "0.1.10" %}
+{% set version = "0.2.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/blambright/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
-  sha256: a5d561ef757d1751766674fba977e339cb4028b057a350969224403f286c4751
+  sha256: 95f7666929cc34efbbd9e0640a13f507b8ec35ec34a9604017cf96c24f7f9ea6
 
 build:
-  noarch: python
   number: 0
+  skip: true  # [s390x]
 
 requirements:
   host:
     - python >=3.6
```

Comment:
This should not have version pinnings. Use `skip` instead.

Reply:
Inflection is a package not available for s390x within defaults and causes a failure to build on this platform. See [here](https://anaconda.org/main/inflection).

Reply:
Could you add a comment that `inflection` is not compatible with `s390x`?

Reply:
Absolutely! Thanks Marco

Reply:
I would be more explicit and say that `inflection` cannot be built on s390x. If it's just not available, it should be built on s390x instead.

Also, the python pinning still needs to be removed. We need to use the `skip` field to limit python versions.